README.md

```diff
@@ -3,27 +3,20 @@
 Hooks, Context Providers, and Components that make it easy to interact with
 Firebase.
 
-> If you're looking for docs for the _deprecated_ ReactFire v1 (the one that
-> uses mixins), click
-> [here](https://github.com/FirebaseExtended/reactfire/tree/v1.0.0)
-
-**Status: Alpha**. ReactFire is meant for React Concurrent Mode, which is only
+⚠️ **Status: Alpha**. ReactFire is meant for React Concurrent Mode, which is only
```

Comment:
How about "status: experimental". We should also add that "we will aim to be API Stable, however"